### ISSUE TITLE
Math

### ISSUE THREAD

**Issue by YouSour** [pacifio/byte_flow#3]
hi , @pacifio do you have any plan to create Math like lodash ? [https://lodash.com/docs/4.17.15#ceil](url)

**Comment by pacifio** [comment]
Yes , thanks for the suggestion , this might come after the null safety update !

**Comment by YouSour** [comment]
@pacifio noted ! 😄
